input/cql/DepressionRemissionatTwelveMonthsFHIR.cql

```diff
@@ -71,11 +78,17 @@ define "SDE Sex":
 define "Denominator Identification Period":
   Interval[start of "Measurement Period" - 14 months, start of "Measurement Period" - 2 months )
 
+/*
+@QICore6Uplift: Update 'Observation' to 'ObservationScreeningAssessment'
+*/ 
 define "Depression Assessments Greater than 9":
   [Observation: "PHQ 9 and PHQ 9M Tools"] DepressionAssessment
     where DepressionAssessment.value > 9
       and DepressionAssessment.status in { 'final', 'amended', 'corrected' }
 
+/*
+@QICore6Uplift: Update both 'Condition' resources to 'ConditionProblemsHealthConcerns'
+*/ 
```

Comment:
Should confirm clinical intent, but as written this should also be a union of ConditionProblemsHealthConcerns and ConditionEncounterDiagnosis